README.md

```diff
@@ -8,7 +8,7 @@ A one-click Ionic app template for Heroku
 Make sure you have [Node.js](http://nodejs.org/) and the [Heroku Toolbelt](https://toolbelt.heroku.com/) installed.
 
 ```sh
-git clone git@github.com:driftyco/ionic-heroku-button.git # or clone your own fork
+git clone https://github.com/driftyco/ionic-heroku-button.git # or clone your own fork
```

Comment:
Why would you want to use https instead of ssh?

Reply:
@maxdignan Sorry for not seeing this earlier. Because running `git clone git@github.com:driftyco/ionic-heroku-button.git` throws a permission denied error.